acceptance/framework/connhelper/connect_helper.go

```diff
@@ -170,6 +175,18 @@ func (c *ConnectHelper) DeployClientAndServer(t *testing.T) {
 		})
 }
 
+func (c *ConnectHelper) SetupNamespace(t *testing.T, namespace string) {
```

Comment:
**suggestion:** I might consider naming this from SetupNamespace to CreateNamespace so that it matches the kubectl command.  (might help with searching?)